mlem/core/metadata.py

```diff
@@ -74,12 +75,13 @@ def save(
         sample_data,
         params=params,
     )
+    path = str(path) if isinstance(path, pathlib.Path) else path
```

Comment:
```suggestion
    path = os.fspath(path)
```

See [`os.fspath`](https://docs.python.org/3/library/os.html#os.fspath) and [PEP-0519](https://peps.python.org/pep-0519).

Reply:
Thanks again